x-pack/platform/plugins/shared/cases/server/routes/api/cases_ai/case_prompt_builder/obs/obs_case_summary_prompt.ts

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { Case } from '../../../../../../common';
+import type { Attachments } from '../../../../../../common/types/domain';
+import { AttachmentType } from '../../../../../../common/types/domain';
+import { PageAttachmentType, PersistableAttachmentType } from '../types';
+
+export function buildObsCaseSummaryPrompt(caseData: Case): string {
+  const { comments = [] } = caseData;
+
+  let prompt = '';
+
+  prompt += buildCaseActivitySummary(comments);
+  prompt += getAnalysisInstructions();
+
+  return prompt;
+}
+
+function getAnalysisInstructions() {
+  return `## Analysis Instructions\n
```

Comment:
@benakansara I curious to learn how did you come up with this prompt. Is there any prompt tool available for this purpose to help you get started?